git-secrets

```diff
@@ -69,7 +69,9 @@ scan() {
 }
 
 scan_history() {
-  local to_scan=$(git rev-list --all)
+  local patterns=$(load_patterns | tr ' ' '|')
+  [ -z "${patterns}" ] && return 0
+  local to_scan=$(git log --all -S"${combined_patterns}" --pickaxe-regex --pretty=%H)
   local output=$(git_grep "" "${to_scan}")
```

Comment:
Exactly, I did not find another way to create an output consistent with `--scan`.